src/commands/root.command.ts

```diff
@@ -48,7 +48,11 @@ export class Root extends CommandRunner {
     super()
   }
 
-  async run(): Promise<void> {}
+  async run(_, options): Promise<void> {
+    if (!(options.version || options.verbose || options.quiet)) {
```

Comment:
No need to add `options.noLog` here?

Reply:
No.  `options.noLog` is not necessary in that `if` statement. Honestly, though I had a reason for doing it this way.

This line is just to make sure that the help doesn't show in situations where a command will actually be ran, and no-log isn't needed to get the desired behavior.